internal/cmd/gocdk/internal/static/_assets/demo/demo_docstore.go

```diff
@@ -135,9 +140,6 @@ const (
       <br/>
       <input type="text" name="value" value="{{ .Value }}">
     </label></p>
-    {{if not .Create}}
-      <p>Revision: {{ .Revision }}</p>
```

Comment:
I dropped the revision stuff from the demo, as it adds more complication.

I still needed to keep `DocstoreRevision` in the `MyDocument` struct, because docstore is still adding that field automatically and the `Get` fails if it's not in the struct (Tuo is fixing this behavior, but it's not in yet).

A possible alternative for later is to make the demo include revisioning, which would include grabbing the revision from the document when initializing the "edit" form, putting it in a "hidden" field in the form, and then putting it back on the `MyDocument` before calling `Replace`.